scripts/genpinctrl/stm32-pinctrl-config.yaml

```diff
@@ -219,6 +219,10 @@
   drive: open-drain
   bias: pull-up
 
+- name: UART_DE / USART_DE / LPUART_DE
```

Comment:
Would you mind adding the same for F1 template ?

Reply:
The F1 and F2 SoCs don't seem to support/have RS-485 configuration.